content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -32,9 +32,9 @@ Users need to be aware of the following when using this feature:
 * Cloning support is only available for dynamic provisioners.
 * CSI drivers may or may not have implemented the volume cloning functionality.
 * You can only clone a PVC when it exists in the same namespace as the destination PVC (source and destination must be in the same namespace).
-* Cloning is only supported within the same Storage Class.
-    - Destination volume must be the same storage class as the source
-    - Default storage class can be used and storageClassName omitted in the spec
+* Cloning is supported with a different Storage Class.
+    - Destination volume should not be the same storage class as the source.
```

Comment:
What???

Reply:
@tengqm please find the [PR](https://github.com/kubernetes-csi/external-provisioner/pull/699) for refrence.

Reply:
The wording is wrong. The destination is allowed to be a different StorageClass to the source, but they are also allowed to be the same StorageClass.

@Kartik494 you need to fix the wording so that it is correct.

Reply:
@sftim for the clarification should i add both  the statements i.e `different StorageClass to the source` and `same StorageClass to the source` ?

Reply:
Maybe `Destination volume can be the same or a different storage class as the source.` ?

Reply:
@tengqm Sure that looks ok to me . I will update accordingly.

Reply:
@tengqm Addressed the comments PTAL .
Thanks!!